wsgi.py

```diff
@@ -10,21 +10,59 @@
 from flask.logging import default_handler
 import requests
 
+from publish_json_schema import PublishJSONSchema
+
 application = Flask(__name__)  # noqa
 
 # Set up logging
 ROOT_LOGGER = logging.getLogger()
 ROOT_LOGGER.setLevel(application.logger.level)
 ROOT_LOGGER.addHandler(default_handler)
 
+VERSION = "0.0.1"
+
 # Upload Service
 UPLOAD_SERVICE_ENDPOINT = os.environ.get('UPLOAD_SERVICE_ENDPOINT')
 
 
-@application.route("/", methods=['POST'])
-def wake_up():
+def validate(input_data)-> dict:
+    """Validate input json."""
+    schema = PublishJSONSchema()
+    return schema.load(input_data)
+
+
+@application.route('/', methods=['GET'])
+def get_liveliness():
+    """Endpoint for Liveliness."""
+    return jsonify(
+        status='OK',
+        message='AIOPS Publisher is up and running!'
+    )
+
+
+@application.route('/api/v0/version', methods=['GET'])
+def get_version():
+    """Endpoint for getting the current version."""
+    return jsonify(
+        status='OK',
+        version=VERSION,
+        message='AIOPS Publisher Version 0.0.1'
+    )
+
+
+@application.route("/api/v0/publish", methods=['POST'])
+def post_publish():
     """Endpoint for upload and publish requests."""
     input_data = request.get_json(force=True)
+
+    validation = validate(input_data)
```

Comment:
Here's the call to the validate function which I think is redundant, because it's just this sequence in disguise:
 
```python
input_data = request.get_json(force=True)
schema = PublishJSONSchema()
validation = schema.load(input_data)
```

I think it's better inlined since the function is only used once. I would also suggest to have the `PublishJSONSchema()` instance stored as a global constant. It's static.

I've also noticed, the workflow for `schema.loads()` will change in new marshmallow version (v3.0 line)
Because [according to the docs](https://marshmallow.readthedocs.io/en/latest/api_reference.html#marshmallow.Schema.loads) it will return a `dict` of the parsed data. It will not have a property `errors`, instead it will raise `ValidationError` exception.

I think we should lock the `marshmallow` to the v2 line in our `Pipfile` for now and place a notice here that this code has to be updated in the future. What do you think? I mean, it's not a problem now, but when `marshmallow` v3 is released it's gonna introduce a bug in new deployments (when the `Pipfile.lock` is regenerated)...

Reply:
`PublishJSONSchema()` instance is now a global constant  - good suggestion! cdc72d8

Also locked the major version of `marshmallow` to `2`